core-plugins/docs/Joiner-batchjoiner.md

```diff
@@ -108,6 +108,14 @@ stage A is the skewed input stage.
 
 For more information about Distribution and data skew, please see the **Skew** section of this documentation.
 
+**Input with Larger Data Skew** Skewed joins can cause a lot of shuffling and disk operation in BigQuery, which impacts query performance.
+BigQuery has some general recommendations when executing join operations on Skewed data sets which can be found here
```

Comment:
can you add a comma before "which"? Also, data sets should be datasets? I think we use the single word.